resources/Resource.h

```diff
@@ -1,3 +1,4 @@
+#pragma once
```

Comment:
this is a generated file, please remove this pragma

Reply:
The pragma is now removed from this file